### DISCUSSION TITLE
How to use Linguard CLI?

### DISCUSSION THREAD

**Discussion by HALKRON** [joseantmazonsb/linguard#106] [Q&A]
I am very interested in this project. The Web GUI is amazing but one downside is that it doesn't correctly detect any wireguard interfaces created outside of the GUI such as making an interface via "ip link" comment. The reason this is important for me because I use Ansible to push configurations to many Wireguard Servers and it would be tedious for me make a peer in 3 different server, scale that to 10 or 20 users and it becomes a problem.

So I search around the Linguard docs to see if there are any CLI options, and in the in-depth section, there are several arguments for Linguard but when I tried putting linguard command in the server CLI, it just says it couldn't find the command. I hope others knows a way for this. 

Source: https://linguard.readthedocs.io/en/stable/in-depth.html#in-depth

**Comment by joseantmazonsb** [comment]
Hello there, there is no Linguard CLI at the moment. However, I'll take this as a feature request.

On the other hand, I don't quite understand your issue regarding the automatic detection of interfaces. Could you clarify the behaviour you were expecting?

**Reply by HALKRON** [reply]
Ah, sorry. There was a typo in my original post. So, based on my experience, whenever I create a new Wireguard interface via CLI, Linguard doesn't know the Wireguard interface, but it DOES show that it is a Wireguard interface however everything is "unknown" in the GUI.

**Reply by joseantmazonsb** [reply]
I see. Well, right now Linguard is unable to manage interfaces created directly through the Wireguard CLI. Importing this kind of interfaces into Linguard, as well as adding a Linguard CLI, is a planned feature which will be included in the next version
